### PR TITLE
[TR] PIM-4187 save and back to grid

### PR DESCRIPTION
**Pull request by jmleroux**

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/save-and-back.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        mediator,
+        BaseForm,
+        Navigation,
+        Routing,
+        messenger
+    ) {
+        return BaseForm.extend({
+            className: 'btn-group',
+            configure: function () {
+                this.parent.extensions['save-buttons'].addButton({
```
Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/save-and-back.js]: you should test if the extension exists before calling a method on it :)

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/save-and-back.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                        'click .save-product-and-back': _.bind(this.saveAndCreate, this)
+                    }
+                });
+
+                return BaseForm.prototype.configure.apply(this, arguments);
+            },
+            saveAndCreate: function () {
+                this.parent.extensions.save.save({ silent: true }).done(_.bind(function () {
+                    messenger.addMessage(
+                        'success',
```
Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/save-and-back.js]: what if a exception occures during saving ?

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/save-and-back.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        'jquery',
+        'underscore',
+        'oro/mediator',
+        'pim/form',
+        'oro/navigation',
+        'routing',
+        'oro/messenger'
+    ],
+    function ($,
+              _,
```
Comment (filipsalpe) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/save-and-back.js]: this indentation looks weird, no?

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/sequential-edit.js [hunk 4]
```diff
@@ -54,6 +54,10 @@ define(
                 );
             },
             addSaveButton: function () {
+                if (!('save-buttons' in this.parent.extensions)) {
+
```
Comment (filipsalpe) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/sequential-edit.js]: extra blank line

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/save-and-back.js (hunk 3 above)
Comment (jmleroux) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/save-and-back.js]: Got to find the good PHPStorm code style for JS

**Comment by sumbobyboys** [comment]
Nice PR ! GTM :)

**Pull request merged**